CMake/gkd350h_defs.cmake

```diff
@@ -1,6 +1,10 @@
 set(NONET ON)
 set(USE_SDL1 ON)
 
+# Do not warn about unknown attributes, such as [[nodiscard]].
+# As this build uses an older compiler, there are lots of them.
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-attributes")

```

Comment:
Maybe also apply this to the amiga builds as its also spewing out warnings about unknown attributes.

Reply:
https://github.com/diasurgical/devilutionX/pull/2009